nvflare/dashboard/cli.py

```diff
@@ -0,0 +1,112 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import os
+import signal
+
+import docker
+import requests
+
+from nvflare.lighter.utils import generate_password
+
+
+def start(port, folder, app_url, app_dest_filename, dashboard_image):
+    if not folder:
+        folder = os.getcwd()
+    environment = dict()
+    if not os.path.exists(os.path.join(folder, "db.sqlite")):
```

Comment:
what is this "db.sqlite"? why we don't need to generate password if it exists

Reply:
This is the sqlite database file.  If it exists, that means project admin is created previously and stored in the database.  Project admin can login the dashboard with the email address and the randomly generated password.